geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js

```diff
@@ -1299,7 +1299,11 @@ const MainController = function(
           // set current theme for legacy components
           this.appTheme_.setCurrentTheme(theme.name);
           // set 'data-theme' attribute for legacy component colors
-          document.getElementsByTagName('body')[0].setAttribute('data-theme', theme.name)
+          document.getElementsByTagName('body')[0].setAttribute('data-theme', theme.name);
+
+          try {
+            $scope.$apply(); // Force apply on theme switcher to update theme name
```

Comment:
I just noticed that `$scope.$digest()` within a try catch does not cause an error in the console, but also seems to solve the issue :slightly_smiling_face: 
```suggestion
            $scope.$digest(); // Force apply on theme switcher to update theme name
```